internal/tools/preview-api-version-linter/historical-exceptions.yml

```diff
@@ -0,0 +1,162 @@
+# Exceptions for historical uses of preview API before the linter tool is setup
+# DO NOT ADD NEW ENTRY
+
+- module: github.com/Azure/azure-sdk-for-go
+  service: resources
+  version: 2021-06-01-preview
```

Comment:
should we consider narrowing this down by packages or resources as well? having this scoped at the version could mean someone could use a preview api that is already in use the service, but for a different resource